src/Publisher.ts

```diff
@@ -324,13 +330,17 @@ export default class Publisher {
             try{
                 const block = queryBlock[0];
                 const query = queryBlock[1];
-
+                const {isInsideCallout, finalQuery} = this.sanitizeQuery(query)
                 const div = createEl('div');
                 const component = new Component();
-                await dvApi.executeJs(query, div, component, path)
+                await dvApi.executeJs(finalQuery, div, component, path)
                 component.load();
-                
-                replacedText = replacedText.replace(block, div.innerHTML);                
+                let replacementString = div.innerHTML
+                if (isInsideCallout) {
+                    const tmp = replacementString.split("\n");
+                    replacementString = " " + tmp.join("\n> ")
```

Comment:
Why are you adding a space here? Wouldn't replacementString.split("\n").join("\n>") be enough, or am i missing something?

Reply:
you're right, latest commit should do the trick